packages/gatsby-cli/src/structured-errors/construct-error.ts

```diff
@@ -18,7 +18,7 @@ const constructError = ({
     text: result.text(otherDetails.context),
     stack: otherDetails.error
       ? sanitizeStructuredStackTrace(stackTrace.parse(otherDetails.error))
-      : null,
+      : [],
```

Comment:
better to have a consistent type of `Error[]`

Reply:
This changes semantics - not having stack trace vs stack trace with 0 frames (it's not the same thing). This is also undocumented/public API so changing "fallback" can impact consumers (we emit this via IPC)

Reply:
let me explore it to see if it's safe or not.

Reply:
empty array is truthy so if someone did check like this it would potentially change beavhour for them

Reply:
Discussing with cloud team here: https://gatsbyjs.slack.com/archives/CCELDN9MM/p1589907510278700